### PR TITLE
Add kafka-manager 1.3.0.8

### PR DESCRIPTION
**Pull request by sheepkiller**

### CODE REVIEW

File: repo/packages/K/kafka-manager/0/marathon.json.mustache [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  "cpus": {{kafka-manager.cpus}},
+  "mem": {{kafka-manager.mem}},
+  "instances": {{kafka-manager.instances}},
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{resource.assets.container.docker.kafka-manager-docker}}",
+      "forcePullImage": true,
+      "network": "BRIDGE",
+      "portMappings": [{ "containerPort": 9000, "protocol": "tcp" }]
```
Comment (pyronicide) [inline repo/packages/K/kafka-manager/0/marathon.json.mustache]: Don't you want a host port?

File: repo/packages/K/kafka-manager/0/marathon.json.mustache [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      "intervalSeconds": 30,
+      "maxConsecutiveFailures": 0,
+      "path": "/",
+      "portIndex": 0,
+      "protocol": "HTTP",
+      "timeoutSeconds": 5
+    }
+  ],
+  "ports": [0],
+  "healthChecks": [
```
Comment (pyronicide) [inline repo/packages/K/kafka-manager/0/marathon.json.mustache]: Duplicate.

File: repo/packages/K/kafka-manager/0/marathon.json.mustache [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      "portIndex": 0,
+      "timeoutSeconds": 30,
+      "maxConsecutiveFailures": 3
+    }
+  ],
+  "env": {
+    "ZK_HOSTS": "{{kafka-manager.zk}}",
+    "KM_ARGS": "-Dkafka-manager.base-zk-path=/{{service.name}}"
+    {{#kafka-manager.application_secret}},
+    "APPLICATION_SECRET": "{{kafka-manager.application_secret}}"
```
Comment (pyronicide) [inline repo/packages/K/kafka-manager/0/marathon.json.mustache]: If the secret is required, any reason to have the if block?

File: repo/packages/K/kafka-manager/0/config.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        "mem": {
+            "description": "Memory to allocate",
+            "type": "number",
+            "default": 1024.0,
+            "minimum": 1024.0
+        },
+        "zk": {
+            "type": "string",
+            "description": "Zookeeper URL for Kafka-manager",
+            "default": "master.mesos:2181"
```
Comment (pyronicide) [inline repo/packages/K/kafka-manager/0/config.json]: We've got a new, better zk url:

    zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181

**Comment by sheepkiller** [comment]
Hi Thomas,

Thanks for your review. Please note I also remove `ports` definition. I think it is now redundant with `hostPort`.

**Comment by pyronicide** [comment]
@sheepkiller looks good, please rebase and I'll merge.

**Comment by sheepkiller** [comment]
Hi @pyronicide,

Done, thanks!

**Comment by pyronicide** [comment]
This is so painful .... One more time (because of the shared metadata, gotta merge in order).

**Comment by sheepkiller** [comment]
no problem. thank you @pyronicide

**Comment by sheepkiller** [comment]
thank you, @pyronicide.

**Pull request merged**